### PR TITLE
Update references to Dandelion’s GitHub account

### PR DESCRIPTION
**Pull request by wchargin**
Summary:
Their primary account is now @decentralion. The repository link is
forwarded by GitHub, but it’s better for to link to the canonical
address.

Test Plan:
One link is modified; clicking the link works.

Running `git grep -i 'dan\(delion\)\?mane'` returns no results.

wchargin-branch: decentralionize

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ through setting up and using TensorBoard. There's an associated
 
 [this video tutorial]: https://www.youtube.com/watch?v=eBbEDRsCmv4
 
-[tutorial with an end-to-end example of training TensorFlow and using TensorBoard]: https://github.com/dandelionmane/tf-dev-summit-tensorboard-tutorial
+[tutorial with an end-to-end example of training TensorFlow and using TensorBoard]: https://github.com/decentralion/tf-dev-summit-tensorboard-tutorial
```
Comment (wchargin) [inline README.md]: Do we want to move this repository into the @tensorflow organization?

Comment (decentralion) [reply README.md]: That would be great; that way it can actually be maintained.
Do you want me to just transfer the repository to @tensorflow?

Comment (wchargin) [reply README.md]: Yes; that’d be great. Thanks!

Comment (wchargin) [reply README.md]: Please leave a comment here when you initiate the transfer.

Comment (martinwicke) [reply README.md]: Yes, add me as admin to the repo and I can transfer it.

Comment (decentralion) [reply README.md]: Sent you an invite link.

Comment (martinwicke) [reply README.md]: Accepted. Still need admin for the repo, then I can transfer it, I think.

Comment (wchargin) [reply README.md]: Retaining `martinwicke/tf-dev-summit-tensorboard-tutorial`.

(Googlers, see <http://b/120805989>.)

**Pull request merged**